connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -1,6 +1,10 @@
+import fs from "fs/promises";
 import { GaxiosError, GaxiosResponse } from "googleapis-common";
 import StatsD from "hot-shots";
+import os from "os";
 import PQueue from "p-queue";
+// @ts-expect-error no types provided for pdf-to-text
+import pdfUtil from "pdf-to-text";
```

Comment:
Did you look at definitelytyped ? `@types/pdf-to-text`
Otherwise I would be in favor of adding our own type stubs, not a fan of having expect error on an import…

Reply:
No existing types @types. I will write our own.